pom.xml

```diff
@@ -917,6 +914,26 @@
       </properties>
     </profile>
     <profile>
+      <id>scala-2.10</id>
+      <activation>
+        <property><name>!scala-2.11</name></property>
+      </activation>
+      <properties>
+        <scala.compat.version>2.10</scala.compat.version>
+        <scala.version>2.10.4</scala.version>
+      </properties>
+    </profile>
+    <profile>
+      <id>scala-2.11</id>
+      <activation>
+        <property><name>scala-2.11</name></property>
+      </activation>
+      <properties>
+        <scala.compat.version>2.11</scala.compat.version>
+        <scala.version>2.11.8</scala.version>
```

Comment:
We need to cap at 2.11.2 to maintain breeze compatibility. see https://github.com/apache/mahout/pull/326/files#diff-5d8d832c3c4016dd77a9c3d92a2c5b69